### PR TITLE
Todo list app backend

### PR DESCRIPTION
**Pull request by sahilrawat001**
# Hackathon Submission: <PROJECT_NAME>

## Name of Team Lead
_Enter your full name_

## GitHub handles of Team Members  
_Enter the GitHub handles of all your team members (including yourself) as separate bullet points_

### Are all of your team members registered on our [Hackathon Signup page](https://apwr.dev/htf24-hackathon)?
_This is a mandatory step for all participants to be considered eligible for judging_

- [x] Yes
- [ ] No

---

## Rules and Code of Conduct  
By submitting this PR, you agree to follow our Rules and Code of Conduct.

- [x] Yes

---

## Anything Else?  
_Any other feedback, queries or information you would like to share with us?_

### CODE REVIEW

**Comment by sahilrawat001** [comment]
done with the changes

File: submissions/sahil_todo.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+I used Appwrite Storage to...
+
+.
+.
+.
+
+-->
+
+## Project Repo  
+[_Share a public repo link of your project_](https://github.com/sahilrawat001/app-write)
```
Comment (adityaoberai) [inline submissions/sahil_todo.md:70]: Please add the repo link to a separate line

File: submissions/sahil_todo.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+<!--
+
+https://github.com/code-capture/CodeCapture-Xamarin
+
+-->
+
+## Demo Video/Photos/Link
+_Share a 2-3 minute demo video or photos or a deployed link of your project_
```
Comment (adityaoberai) [inline submissions/sahil_todo.md:79]: It is mandatory to submit a demo video @sahilrawat001

**Review by adityaoberai — CHANGES_REQUESTED** [review]

**Comment by adityaoberai** [comment]
@sahilrawat001 I am accepting this PR because it has a valid project, however, not having a demo will be penalized in judging

**Review by adityaoberai — APPROVED** [review]

**Pull request merged**